docs/scss/_base/_base.scss

```diff
@@ -57,7 +57,7 @@ h2,
 h3,
 h4 {
   font-family: 'RedHatDisplay', 'Overpass', Helvetica, sans-serif;
-  line-height: 1.4;
+  line-height: 1.3;
```

Comment:
if we're anyways touching these rules can we advance to using tokens?

Reply:
For sure. I'm hoping to get this PR merged, so our docs look more like our mockups asap. I've opened a new issue (#937) to switch over to tokens though.